model_analyzer/config/generate/model_config_generator.py

```diff
@@ -160,23 +164,22 @@ def _make_remote_model_config(self):
 
         return model_config
 
-    def _is_default_config_in_configs(self, configs):
-        return {} in configs
+    def _is_default_combo_in_param_combos(self, param_combos):
+        return self._default_param_combo in param_combos
 
-    def _add_default_config(self, configs):
-        # Add in an empty configuration, which will apply the default values
-        configs.append({})
+    def _add_default_combo(self, param_combos):
+        # Add in an empty combo at the start of the list, which will just apply the default values
+        #
+        param_combos.insert(0, self._default_param_combo)
 
     def _finalize_configs(self, configs):
         for config in configs:
-            model_tmp_name = self._base_model_name
-            if not self._remote_mode:
-                model_tmp_name = self._get_model_variant_name()
-
-            config.set_field('name', model_tmp_name)
             config.set_cpu_only(self._base_model.cpu_only())
 
-    def _get_model_variant_name(self):
-        variant_name = f'{self._base_model_name}_config{self._model_name_index}'
-        self._model_name_index += 1
+    def _get_model_variant_name(self, param_combo):
+        if param_combo is self._default_param_combo:
+            variant_name = f'{self._base_model_name}_config_default'
+        else:
+            variant_name = f'{self._base_model_name}_config_{self._model_name_index}'
```

Comment:
@debermudez I am adding the underscore between "config" and the # (that @nv-braf requested in your review) now. That way the format is consistent with the new config_default. Is there anything else that needs to change (documentation?)

Reply:
- qa/L0_profile/check_results.py
- qa/L0_quick_start/test.sh
- tests/test_model_config.py
- tests/test_run_config_generator.py
- model_analyzer/config/run/run_config_generator.py
- model_analyzer/triton/model/model_config.py

Those would be all the files to touch.

Reply:
run_config_generator change: https://github.com/triton-inference-server/model_analyzer/pull/281/files#diff-9f379e5c26489133c3fdf98fabfc766b3845fc0f9164f920b525750b57c17995R133 
I think you are okay on this one.

test_run_config_generator: instances of the name vgg_16_graphdef_config0 should have to change to vgg_16_graphdef_config_0 right?

Reply:
I don't see that anywhere in test_run_config_generator or anywhere in our repo

Reply:
must have changes since i made the first upgrade. Lets resolve this one then.